src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -52,6 +53,11 @@
     private $updatePage;
 
     /**
+     * @var HistoryPageInterface
+     */
+    private $historyPage;
```

Comment:
Can be a case for separate PR, but I would say to do it right now. I am afraid that we will forget about it and end up with too big class